Source/WebKit/Platform/cocoa/LayerHostingContext.mm

```diff
@@ -60,13 +68,20 @@
 #if PLATFORM(IOS_FAMILY) && !PLATFORM(MACCATALYST)
     // Use a very large display ID to ensure that the context is never put on-screen
     // without being explicitly parented. See <rdar://problem/16089267> for details.
-    layerHostingContext->m_context = [CAContext remoteContextWithOptions:@{
+    auto contextOptions = @{
         kCAContextSecure: @(options.canShowWhileLocked),
 #if HAVE(CORE_ANIMATION_RENDER_SERVER)
         kCAContextIgnoresHitTest : @YES,
         kCAContextDisplayId : @10000
 #endif
-    }];
+    };
+#if USE(EXTENSIONKIT)
+    if (options.useHostable && [get_SEHostableClass() respondsToSelector:@selector(createHostableWithOptions:error:)]) {
+        layerHostingContext->m_hostable = [get_SEHostableClass() createHostableWithOptions:contextOptions error:nil];
```

Comment:
This is a leak: assigning to a `RetainPtr<>` from a function that presumably returns a retained value